### PR TITLE
Cleanup after rebasing on py.test and python 3 changes

### PR DESCRIPTION
**Pull request by pschella**
Inverse PR

### CODE REVIEW

File: python/lsst/daf/base/baseLib.py [hunk 1]
```diff
@@ -85,18 +135,23 @@ def _propertyContainerAdd(container, name, value, typeMenu, *args):
...
         if isinstance(exemplar, checkType):
             return getattr(container, "add" + typeMenu[checkType])(name, value, *args)
     raise lsst.pex.exceptions.TypeError("Unknown value type for %s: %s" % (name, t))
 
+
 # Mapping of type to method names
+from past.builtins import long
 _PS_typeMenu = {bool: "Bool",
                 int: "Int",
-                long: "LongLong",
```
Comment (timj) [inline python/lsst/daf/base/baseLib.py]: If `long` is being removed from the list completely how will this work on Python 2? (and why have the `long` import if it's been removed).

Comment (pschella) [inline python/lsst/daf/base/baseLib.py]: Hmm, strange. It seemed to work fine. But I put it back now.

Comment (timj) [inline python/lsst/daf/base/baseLib.py]: Maybe none of your converted code is using `long` type?

**Comment by jdswinbank** [comment]
Obsolete.

**Pull request closed without merging**